docs/admin_menu/admin_menu.rst

```diff
@@ -556,9 +556,86 @@ Storages
 --------
 
 On STORAGES tab, you can see what kind of mount volumes (usually NFS) exist.
+From 23.03 version, We provide per-user/per-project quota setting on stoarge that supports quota management.
+By using this feature, admin can easily manage and monitor the exact amount of storage usage for each user and project based folder.
 
 .. image:: storage_list.png
 
+In order to set quota, you need to first access to storages tab in resource page.
+And then, click gear icon in control column.
+
+.. note:: 
+   Please remind that quota setting is only available in storage that provides quota setting 
+   (e.g. XFS, CephFS, NetApp, Purestorage, etc.). Although you can see the usage of storage 
+   in quota setting page regardless of storage, you cannot configure the quota which doesn't 
+   support quota configuration internally.
+   
+   .. image:: no_support_quota_setting.png
+
+Quota Setting Panel
+~~~~~~~~~~~~~~~~~~~~
+
+In Quota setting page, There are two panels that represent the corresponding items for each panel's title.
+
+.. image:: quota_setting_page.png
+
+* Resource
+   * Usage: Shows the actual amount usage of the selected storage. 
+   * Endpoint: Represents the mount point of the selected storage.
+   * Backend Type: the type of storage.
+   * Capabilities: the supported feature of the selected storage.
+
+* Quota Settings
+   * For User: Configure per-user quota setting here.
+   * For Project: Configure per-project quota(project-folder) setting here.
+   * ID: Corresponds to user or project id.
+   * Hard Limit (GB): Currently set hard limit quota for selected quota.
+   * Control: Provides editing the hard limit or even deleting the quota setting.
+
+
+Set User Quota
+~~~~~~~~~~~~~~~
+
+In Backend.AI, there are two type of vfolders created by user and admin(project). In this section,
+we would like to show how to check current quota setting per-user and how to configure it.   
+First, Make sure the active tab of quota settings panel is ``For User``. Then, Select user you desire to 
+check and edit the quota. You can see the quota id that corresponds to user's id and the configuration already set 
+in the table, if you already set the quota.   
+
+.. image:: per_user_quota.png
+
+Of course, you want to edit the quota, Simply clicking ``Edit`` button in 
+Control column will do. After Clicking ``Edit`` button, you may see the small modal that enables configuring quota setting. 
+After input the exact amount, don't forget to Click ``OK`` button, unless the changes will not be applied.
+
+.. image:: quota_settings_panel.png
+
+
+Set Project Quota
+~~~~~~~~~~~~~~~~~~
+
+Setting quota on project-folder are similar to setting user quota. The difference between setting project quota and user quota is 
+that to confirm and setting the project quota requires one more procedure, which is selecting the domain that project is dependent.
+The rest of things are same.
```

Comment:
Suggestion (grammar)

Setting a quota on project-folder is similar to setting a user quota. The difference between setting project quota and user quota is to confirm setting the project quota requires one more procedure, which is selecting the domain that the project is dependent on. The rest are the same.